packages/bits/src/lib/selector/selector.component.html

```diff
@@ -25,7 +24,6 @@
 <div #popupArea></div>
 
 <nui-overlay #overlay
-             [role]="'menu'"
              [toggleReference]="toggleButton"
```

Comment:
should the roleAttr be specified here?

Reply:
No, because of `nui-menu-popup` inside `nui-overlay`. `nui-menu-popup` under the hood has the role="menu"